services/movie_session.py

```diff
@@ -41,4 +41,16 @@
 
 
 def delete_movie_session_by_id(session_id: int) -> None:
-    MovieSession.objects.get(id=session_id).delete()
+    try:
+        MovieSession.objects.get(id=session_id).delete()
+    except MovieSession.DoesNotExist("There is no movie session with this ID"):
+        pass
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    tickets = Ticket.objects.filter(movie_session_id=movie_session_id)
+    result_list = []
+    for ticket in tickets:
+        dic = {"row": ticket.row, "seat": ticket.seat}
+        result_list.append(dic)
+    return result_list
```

Comment:
Instead of manually creating a list of dictionaries, you could use Django's `values()` method. This method returns a QuerySet that returns dictionaries, rather than model instances, when used as an iterable. Each dictionary represents an object, with the keys corresponding to the attribute names of model objects.